api/app/GraphQL/Validators/Mutation/ChangeApplicationSuspendedAtValidator.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\GraphQL\Validators\Mutation;
+use Database\Helpers\ApiEnums;
+use Illuminate\Validation\Rule;
+use Nuwave\Lighthouse\Validation\Validator;
+
+final class ChangeApplicationSuspendedAtValidator extends Validator
+{
+    /**
+     * Return the validation rules.
+     *
+     * @return array<string, array<mixed>>
+     */
+    public function rules(): array
+    {
+        return [
+            'submitted_at' => ['required'],
+        ];
+    }
+
+    public function messages(): array
+    {
+        return  [
+            'required' => 'Must be submitted',
```

Comment:
Whatever you put as your message should also exist in packages/i18n/src/messages/apiMessages.ts so it can be translated.

Reply:
Added with [9bdb5ca](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5832/commits/9bdb5ca1e69981ef196812cec9fca1febb12744b).

Reply:
Good stuff.  Now the linter will update the need to update the ID for you and you'll need to get the French string from Dasha.

Reply:
Just waiting on this translation now. It should be back soon.